src/FrameBuffer.cpp

```diff
@@ -538,6 +538,11 @@ void FrameBufferList::saveBuffer(u32 _address, u16 _format, u16 _size, u16 _widt
 	if (m_pCurrent != NULL) {
 		// Correct buffer's end address
 		if (!m_pCurrent->isAuxiliary()) {
+			if(m_prevColorImageHeight == 0)
+			{
+				m_prevColorImageHeight = VI.height;
+			}
+			
 			if (gDP.colorImage.height > 200)
```

Comment:
m_prevColorImageHeight is ok to be zero.
The problem is that resulted gDP.colorImage.height is not tested against zero